docs/web3-eth.rst

```diff
@@ -1664,9 +1664,9 @@ Example
     }
 
     var tx = new Tx(rawTx, {'chain':'ropsten'});
-    tx.sign(privateKey);
+    var signedTx = tx.sign(privateKey);
```

Comment:
current example code in our doc works fine, and sign doesn't return any thing, if you refer ethereumjs lib doc. so we dnt need this change/PR.  checked with "ethereumjs-tx": "^2.1.2"

Reply:
That repository was last updated 2 years ago and `ethereumjs-tx` is a deprecated module.  It has been replaced by [@ethereumjs/tx](https://www.npmjs.com/package/@ethereumjs/tx) which has the API represented by my PR.

Reply:
[This](https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/tx/docs/classes/Transaction.md#sign) is the API for the currently supported method.